blocks/bricks/conv.py

```diff
@@ -207,16 +220,105 @@ def apply(self, input_):
             with the last two dimensions downsampled.
 
         """
-        output = max_pool_2d(input_, self.pooling_size, st=self.step)
+        output = max_pool_2d(input_, self.pooling_size, st=self.step,
```

Comment:
(It seems that the eventual solution in Theano will be replacing `max_pool_2d` with something named appropriately)